seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -374,7 +374,7 @@
       "translations": {
         "de": {
           "description": [
-            "Kommentieren erlaubt dir Kommentare innerhalb des Codes zu hinterlassen, ohne diesen selbst zu beeinflussen.",
+            "Kommentieren erlaubt es dir, Bemerkungen innerhalb des Codes zu hinterlassen, ohne diesen selbst zu beeinflussen.",
```

Comment:
I wouldnt change **Kommentare** here, but keep the other change. Only revert the translation into Bermerkungen.

comments have a specific meaning in this context, that can differ from _Bemerkungen_. Comments can also include documentation for example.